src/components/common/app-title-cols.vue

```diff
@@ -1,13 +1,21 @@
 <template>
-  <v-row no-gutters>
-    <v-col cols="12" md="6" lg="6" class="mt-4 pl-2" :class="{'pt-10': !isMobile}">
-      <span class="h2 colour-white">Name Request
-      </span>
-    </v-col>
-    <v-col cols="12" md="6" lg="6">
-      <Stats />
-    </v-col>
-  </v-row>
+  <div>
+    <v-row no-gutters>
+      <v-col cols="12" md="6" lg="6" class="mt-4" :class="{'pt-10': !isMobile}">
+        <span class="h2 colour-white" id="app-title">Name Request
+        </span>
+      </v-col>
+      <v-col cols="12" md="6" lg="6">
+        <Stats />
+      </v-col>
+    </v-row>
+    <v-row no-gutters>
+      <v-col cols="12" md="6" lg="6" class="mb-3" :class="{'pt-3': !isMobile}">
+        <span class="h3 colour-white" id="app-subtitle">Get started with your business
+        </span>
+      </v-col>
+    </v-row>
+  </div>
```

Comment:
Is there a reason you didn't combine the title and subtitle into a single row and column?

Reply:
Never mind -- I see you have 2 elements, and the template requires a single element, so the div is the right solution. Resolving.